### PR TITLE
[16.0][ADD] shopinvader_restapi_sale_packaging: extracted from shopinvader_sale_packaging

### PR DESCRIPTION
**Pull request by marielejeune**
Extensions of restapi `sale` and `cart` services for using packages.
Extracted from `shopinvader_sale_packaging`

### CODE REVIEW

File: shopinvader_restapi_sale_packaging/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Shopinvader Restapi Sale Packaging",
+    "Summary": """
+        REST services take care of if products are sold by packagings.
+    """,
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Camptocamp SA",
```
Comment (simahawk) [inline shopinvader_restapi_sale_packaging/__manifest__.py]:
```suggestion
    "author": "Camptocamp, ACSONE",
```

**Review by simahawk — COMMENTED** [review]
LG

**Review by cyrilmanuel — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]
All the dependencies have been merged.

This needs a rebase and it should be good to go

**Comment by marielejeune** [comment]
I've rebased and removed merged dependencies from test-requirements

**Review by cyrilmanuel — APPROVED** [review]

**Pull request merged**